quinn-proto/src/endpoint.rs

```diff
@@ -677,8 +677,14 @@ impl Endpoint {
         header: &ProtectedInitialHeader,
     ) -> Result<(), TransportError> {
         let config = &self.server_config.as_ref().unwrap();
-        if self.cids_exhausted() || self.incoming_buffers.len() >= config.max_incoming {
-            return Err(TransportError::CONNECTION_REFUSED(""));
+        if self.cids_exhausted() {
```

Comment:
I couldn't think of any problems with revealing this to the client, but maybe others can?

Reply:
Seems fine to me.